src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -691,7 +705,11 @@ public Visitable visit(CatchClause node, Void p) {
     HashSet<String> currentLocalVariables = new HashSet<>();
     currentLocalVariables.add(node.getParameter().getNameAsString());
     localVariables.addFirst(currentLocalVariables);
+    // Since there can not be a catch clause inside a catch clause, we don't need to use a temporary
+    // variable like in the case of insideTargetMethod.
```

Comment:
Do you have evidence for this claim?

I don't see anything in JLS 14.20 that suggests this, and a google search for "nested try-catch java" turns up results that suggest that it is not only possible but somewhat common (even if it's ugly), e.g.:
* https://softwareengineering.stackexchange.com/questions/118788/is-using-nested-try-catch-blocks-an-anti-pattern
* https://softwareengineering.stackexchange.com/questions/237444/alternatives-to-nested-try-catches-for-fallbacks
* etc